servers/rendering/renderer_rd/renderer_storage_rd.cpp

```diff
@@ -5125,7 +5125,8 @@ void RendererStorageRD::particles_collision_set_height_field_resolution(RID p_pa
 	if (particles_collision->heightfield_resolution == p_resolution) {
 		return;
 	}
-
+	
+	ERR_FAIL_INDEX(p_resolution, RS::PARTICLES_COLLISION_HEIGHTFIELD_RESOLUTION_MAX);
```

Comment:
This should happen at the very start of the function, there's no reason to do run the code above to abort here.

Reply:
Note: When doing the changes, please use `git commit --amend` to modify the original commit instead of adding a fixup commit. You can then update the PR with `git push --force`. See [PR workflow](https://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html) for details.

There's also a style issue introduced on line 5128 (trailing whitespace).

Reply:
tyvm for the help!! I added those changes to the pull request :)